src/data_chain/mod.rs

```diff
@@ -60,11 +60,65 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
+    use {GROUP_SIZE, MIN_ADULT_AGE};
     use full_info::FullInfo;
+    use itertools::{self, Itertools};
     use maidsafe_utilities::SeededRng;
+    use routing_table::Prefix;
     use rust_sodium;
+    use serde::Serialize;
     use std::iter;
 
+    /// Creates a collection of `GROUP_SIZE` `FullInfo`s for nodes with ages `MIN_ADULT_AGE`,
+    /// `MIN_ADULT_AGE + 1`, `MIN_ADULT_AGE + 2`, etc.  If
+    /// `in_prefix` is `Some`, all nodes' IDs will fit that prefix.
+    pub fn create_full_infos(in_prefix: Option<Prefix>) -> Vec<FullInfo> {
```

Comment:
Probably it'd be also worth to separate these functions into their own `data_chain::test_utils` module in the future.